woocommerce/woocommerce
# Fix woocommerce admin eslint command & errors

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/woocommerce/issues/32310.

Fixes the woocommerce-admin lint command & errors

- Copied woocommerce-admin's .eslintrc.js to js packages
- Corrected paths in tsconfig
- Ignored core's js packages from woocommerce-admin's eslint
- Fixed JS & SCSS lint errors
- Add lint cmd to nx `project.json`

### How to test the changes in this Pull Request:

- Run `pnpm nx build woocommerce`
- Should run `pnpm nx lint woocommerce-admin --skip-nx-cache` successfully
- Should run `pnpm nx lint:js:packages woocommerce-admin --skip-nx-cache` successfully


### Other information:




### Changelog entry

Dev: Fix woocommerce-admin lint command & errors

### FOR PR REVIEWER ONLY:
